### ISSUE TITLE
Feature request: Ability to update instances via zips

### ISSUE THREAD

**Issue by Rotekoppen** [PolyMC/PolyMC#420]
Ive until now used instances to distribute modpacks among my friends, works brilliantly

However to send updates a huge issue appears, theyll either need to manually extract, or reinstall the instance

What i am hoping for is a right click -> update instance from zip -> select zip file

other possible features is deleting files when updating, or apply a diff to files

future things, or problably up to a fork is autoupdating

**Comment by ZekeZDev** [comment]
We do plan on adding mod updates and I dont think this is a bad idea. #65 #274 are similar requests which I will link with this one. 

Personally (not speaking on behalf of any of the other devs) I think it would make sense we start to phase out the .zip export option and instead use modrinth and curseforge options or mabye a custom solution similar to packwiz.

**Comment by Rotekoppen** [comment]
> I think it would make sense we start to phase out the .zip export option and instead use modrinth and curseforge options or mabye a custom solution similar to packwiz.

As long as it is realitivily easy to edit using external programs and distribute

the best thing would be if the instance export would fit inside 8mib upload limit, this would be possible using modsman but not possible to do nativly. an implementation is even easier now that we have a semi complete mod manager

**Comment by ZekeZDev** [comment]
Yeah exactly I dont think zip files are convinent as soon as you add more then a single mod and modloader your already past the upload limit for most chat services.

**Comment by HalJordan** [comment]
Don't forget unless you're just replacing the instance with the new one, but keeping the world, you'll have to deal with all the duplicate mods.
